lib/datasource/go/digest.ts

```diff
@@ -32,6 +33,9 @@ export async function getDigest(
     case bitbucket.id: {
       return bitbucket.getDigest(source, tag);
     }
+    case gitlab.id: {
+      return gitlab.getDigest(source, tag);
+    }
```

Comment:
i think this needs a new test?

Reply:
Yes, and actually the current spec for `getDigest` is not failing as expected:
https://github.com/renovatebot/renovate/blob/main/lib/datasource/go/digest.spec.ts#L52-L63

I'll fix and add a correct test